song-server/src/test/java/bio/overture/song/server/service/InterruptedDatabaseConnection.java

```diff
@@ -41,7 +43,7 @@
 import static org.junit.Assert.assertTrue;
 
 
-//@Ignore("need to replace this with something that doesnt make system calls")
+@Ignore("need to replace this with something that doesnt make system calls")
```

Comment:
@d8660091 i think you forgot to remove this. Also, change the name of this class to `InterruptedDatabaseConnectionTest.java`. It was not running because the file did not originally contain the suffix Test

https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html